nl/guides/user-management/README.md

```diff
@@ -0,0 +1,119 @@
+:::
+
+## Navigeren naar een gebruiker
+
+Als [lesgever](#lesgever) navigeer je naar
+het [gebruikersoverzicht](#gebruikersoverzicht) van je cursus door naar je cursus te navigeren en vervolgens op <span class="guilabal">Geregistreerde gebruikers</span> te klikken in de cursusbeschrijving.
+
+![admin menu users](./staff.admin_menu_users.png)
+
+## Gebruikersoverzicht
+In het **gebruikersoverzicht** worden de gebruikersnaam, de naam, het
+emailadres en [voortgangsstatistieken](#voortgangsstatistieken) opgelijst uit het
+[gebruikersprofiel](/nl/for-students#gebruikersprofiel)
+van alle gebruikers van jouw cursus. Voor de naam van elke [lesgever](#lesgever) en [beheerder](#beheerder)
+staat een
+[icoontje](#gebruikers-beheersrechten-icoontje) dat correspondeert met de
+[beheersrechten](#beheersrechten) die aan
+de gebruiker [toegewezen](#beheersrechten-instellen) werden.
+
+![image](./staff.users.png)
+
+## Gebruiker zoeken
+Gebruik de zoekbalk bovenaan het
+[gebruikersoverzicht](#gebruikersoverzicht) om te zoeken naar specifieke gebruikers op basis van een
+gebruikersnaam, een naam of een emailadres.
+
+![image](./staff.users_filtered.png)
+
+## Gebruiker selecteren
+Druk op de naam om naar de
+[profielpagina](/nl/for-students#profielpagina) van de
+gebruiker te [navigeren](#gebruiker-navigeren).
+
+![image](./staff.user_profile_link.png)
+
+## Gebruiker voortgangsstatistieken
+De **voortgangsstatistieken** van een gebruiker worden weergegeven middels een vooruitgangsbalkje. Het groen-ingekleurde deel stelt het aandeel [oefeningen](/nl/for-students#oefening) in de cursus waarvoor de gebruiker een *correcte* [oplossing](/nl/for-students#oplossing) voor heeft [ingediend](/nl/for-students#oplossing-indienen). Het rode gedeelte stelt het aandeel oefeningen voor waar de gebruiker reeds oplossingen voor heeft ingediend, maar nog niet correct heeft opgelost. Het grijze gedeelte stelt de oefeningen voor waar de gebruiker nog niet aan begonnen is.
+
+![statistics](./user_progress_statistics.png)
+## Beheersrechten
+
+Om sommige acties te kunnen uitvoeren op Dodona heb je bijkomende
+**beheersrechten** nodig. Als [lesgever](#lesgever) kan je zien in het [gebruikersoverzicht](#gebruikersoverzicht) door te kijken naar onderstaande [icoontjes](#icoontjes-voor-beheersrechten). Je kan dan ook de acties aan de rechterkant gebruiken om de [beheersrechten](#beheersrechten) van een gebruiker in te stellen.
+Als [lesgever](#lesgever) kan je enkel de
+beheersrechten <span class="guilabel">Student</span> en
+<span class="guilabel">Cursusbeheerder</span> toekennen aan gebruikers
+(inclusief jezelf). Op die manier kan je een gebruiker promoveren tot
+cursusbegeleider of degraderen tot [student](#student).
+
+![image](./staff.user_edit_permission.png)
+
+::: tip Belangrijk
+
+Als je jezelf als [lesgever](#lesgever)
+degradeert tot [student](#student), dan
+kan je jezelf daarna niet terug promoveren tot lesgever.
+:::
+
+## Icoontjes voor beheersrechten
```

Comment:
Within the context of a course, only course permissions matter. So I would only talk about course permissions here.

In another section I would mention the global permissions, probably in the course management section  to explain that only teachers can create new courses and repositories.